components/LeagueEntries/LeagueEntries.test.tsx

```diff
@@ -5,30 +5,31 @@ import React from 'react';
 describe('LeagueEntries', () => {
   it(`renders 'default' state without a pick made`, () => {
     render(
-      <LeagueEntries entryName="Entry 1" linkUrl="" userPickHistory={[]} />,
+      <LeagueEntries entryName="Entry 1" isLockedOutProp={false} linkUrl="" userPickHistory={[]} />,
```

Comment:
Where's the test where `isLockedOutProp` is true?

Reply:
I will add this.